main.go

```diff
@@ -97,6 +97,7 @@ var (
 	healthAddr           = flag.String("health-addr", ":9090", "The address to which the health endpoint binds.")
 	metricsAddr          = flag.String("metrics-addr", "0", "The address the metric endpoint binds to.")
 	port                 = flag.Int("port", 443, "port for the server. defaulted to 443 if unspecified ")
+	host                 = flag.String("host", "", "the host address the webhook server listens on. defaults to all addresses.")
```

Comment:
can we add docs for this? perhaps in https://open-policy-agent.github.io/gatekeeper/website/docs/customize-startup?

Reply:
We could. I took a look and `port` is missing as well.

IMO we should also be careful about not documenting *every* flag, since they are also self-documented via the `--help` command, but rather highlight the most interesting flags that users might not know to search for.

Since `host` and `port` are fairly common customizations, should they qualify here?

Reply:
I agree with you for `gator` but for `gatekeeper`, it's not self-documenting as we don't ship `gatekeeper` as a binary and it's not something that users would use directly from their terminal.

Reply:
`docker run openpolicyagent/gatekeeper:v3.10.0-beta.0 --help` prints out flags. Would it be better to document that as a way of getting the full list of options?

Reply:
> `docker run openpolicyagent/gatekeeper:v3.10.0-beta.0 --help` prints out flags. Would it be better to document that as a way of getting the full list of options?

+1 on adding this to the customize-startup doc

Reply:
added